src/main/java/org/openrewrite/kotlin/internal/KotlinPrinter.java

```diff
@@ -84,6 +84,9 @@ public J visitBinary(K.Binary binary, PrintOutputCapture<P> p) {
             case Contains:
                 keyword = "in";
                 break;
+            case Not:
```

Comment:
We'll need to adjust binary types based on what we decide during the discussion.
I think it will make more sense to set the type based on the keywords rather than the desugared syntax.

I.E. `Contains` => `In` and `Not` => `NotIn`.

[Source context](https://kotlinlang.org/docs/operator-overloading.html#in-operator)
`a in b` is translated to `b.contains(a)` and `a !in b` is translated to `!b.contains(a)`.

So it would either be the binary name `NotIn` or the translated name `NotContains`.